### PR TITLE
Created problem_41 in project_euler

### PR DESCRIPTION
**Pull request by Kush1101**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by Kush1101** [comment]
@cclauss , 
I am Requesting a review.

**Comment by Kush1101** [comment]
@cclauss . Please review it

File: project_euler/problem_41/sol1.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    """
+    if n % 2 == 0:
+        return False
+    for i in range(3, int(sqrt(n) + 1), 2):
+        if n % i == 0:
+            return False
+    return True
+
+
+def compute_pandigital_prime(n: int) -> List[int]:
```
Comment (cclauss) [inline project_euler/problem_41/sol1.py:37]: This function does not return one prime but instead returns a list of primes.
```suggestion
def compute_pandigital_primes(n: int) -> List[int]:

```

File: project_euler/problem_41/sol1.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    >>> compute_pandigital_prime(2)
+    []
+    >>> max(compute_pandigital_prime(4))
+    4231
+    >>> max(compute_pandigital_prime(7))
+    7652413
+    """
+    pandigital_str = "".join(str(i) for i in range(1, n + 1))
+    perm_list = [int("".join(i)) for i in permutations(pandigital_str, n)]
+    primes = [num for num in perm_list if is_prime(num)]
```
Comment (cclauss) [inline project_euler/problem_41/sol1.py:49]:
```suggestion
    return [num for num in perm_list if is_prime(num)]
```

File: project_euler/problem_41/sol1.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    return primes
+
+
+if __name__ == "__main__":
+    print(max(compute_pandigital_prime(7)))
```
Comment (cclauss) [inline project_euler/problem_41/sol1.py:54]:
```suggestion
    print(f"{max(compute_pandigital_prime(7)) = }")

```

**Comment by Kush1101** [comment]
I made all the changes. @cclauss .

**Review by cclauss — APPROVED** [review]

**Comment by cclauss** [comment]
Nice!

**Pull request merged**